app/controllers/state_file/questions/other_filing_options_controller.rb

```diff
@@ -0,0 +1,6 @@
+module StateFile
+  module Questions
+    class OtherFilingOptionsController < QuestionsController
+    end
```

Comment:
I would suggest adding and empty edit action here to prevent confusion since on the config and the template file you reference it.